src/Domain/Entity/Event.php

```diff
@@ -32,6 +33,11 @@ class Event
     private ?Organizer $organizer = null;
     private ?Timestamp $lastModified = null;
 
+    /**
+     * @var array<Attendee>
+     */
+    private array $attendee = [];
```

Comment:
For arrays, the plural should be used, because more than a single attendee can be added.

```suggestion
    private array $attendees = [];
```

Reply:
Fixed in commit #54ad96b